src/openms/include/OpenMS/KERNEL/MSSpectrum.h

```diff
@@ -120,6 +123,11 @@ namespace OpenMS
     /// Integer data array vector type
     typedef OpenMS::DataArrays::IntegerDataArray IntegerDataArray ;
     typedef std::vector<IntegerDataArray> IntegerDataArrays;
+
+    /// Intensity type
+    typedef std::vector<IntensityType> IntensityTypes;
```

Comment:
maybe rename to IntensitiesType and PositionsType

Reply:
Ohk I'll try that